src/sentry/api/endpoints/project_transaction_threshold.py

```diff
@@ -78,8 +78,8 @@ def post(self, request, project):
 
         data = serializer.validated_data
 
-        with transaction.atomic():
```

Comment:
oops: https://docs.djangoproject.com/en/3.2/topics/db/transactions/#controlling-transactions-explicitly

Reply:
Lmk if you have any suggestions for a better pattern too